src/configs/chirpstack.js

```diff
@@ -1,6 +0,0 @@
-const {env} = process;
-
-export default {
-    apiUrl: env.CHIRPSTACK_REST_API_URL,
```

Comment:
we really don't need this?

Reply:
it was used by 'ns-webhook/chirpstack' route which not exists anymore